File: programming/java/PrintFile.java
package spc.compsoc.example;

import java.nio.file.Paths;
import java.nio.file.Files;

public class PrintFile{
  public static void main(String[] args){
    try{
      Files.readAllLines(Paths.get("../common/PrintText.txt")).forEach(System.out::println);
    }catch(IOException ioe){
      System.err.println("Unable to read file.");
      ioe.printStackTrace();
    }
  }
}

File: programming/c++/PrintFile.cpp
#include <iostream>
#include <fstream>

int main()
{
    std::ifstream ifs("../_common/PrintText.txt");
    std::string content( (std::istreambuf_iterator<char>(ifs) ),
                       (std::istreambuf_iterator<char>()    ) );
    std::cout << content << std::endl;

    return 0;
}